src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/touchstoneRepository/GetScenarioTests.kt

```diff
@@ -12,15 +12,10 @@ import org.vaccineimpact.api.models.*
 
 class GetScenarioTests : TouchstoneRepositoryTests()
```

Comment:
This class had become a monster, so GetCoverageDataForScenarioTests and GetCoverageDataForResponsibiityTests have been split off from it